akka-docs/src/main/paradox/typed/durable-state/persistence-style.md

```diff
@@ -30,3 +30,14 @@ Scala
 
 Java
 :  @@snip [AccountExampleWithNullDurableState.java](/akka-cluster-sharding-typed/src/test/java/jdocs/akka/cluster/sharding/typed/AccountExampleWithNullDurableState.java) { #account-entity }
+
+## Leveraging Java 21 features
+
+When building durable state entities in a project using Java 21 or newer, the @javadoc[DurableStateOnCommandBehavior](akka.persistence.typed.statejavadsl.DurableStateOnCommandBehavior)
```

Comment:
The whole naming with `OnCommand` is pretty weird, but I don't have any other suggestion than following the track of OnMessage

Reply:
The reason I went with that is that there never is an `onMessage` method here, what you define is `onCommand` and `onEvent`. The choice is if we are inconsistent with this API itself, or with another class further away.

Reply:
I mean that the OnMessage naming is not great to begin with, but I see no other option than continue and here the corresponding would be OnCommand as you have done.